home/api.py

```diff
@@ -47,7 +48,12 @@ def list(self, request, *args, **kwargs):
         super(ContentPagesViewSet, self).list(self, request, *args, **kwargs)
 
     def get_queryset(self):
-        queryset = super(ContentPagesViewSet, self).get_queryset()
+        qa = self.request.query_params.get("qa")
+        queryset = ContentPage.objects.live()
```

Comment:
If we keep the `queryset = super(ContentPagesViewSet, self).get_queryset()`, and do a `queryset = queryset | ContentPage.objects.not_live()`, does that maybe help with being able to include the home and index pages?